app/models/user.rb

```diff
@@ -41,4 +41,13 @@ def card_for_review
   def has_linked_github?
     authentications.where(provider: 'github').present?
   end
+
+  def self.get_review_counters
+    User.joins(:cards)
+        .select("id")
+        .select("email")
+        .select("count(cards.id) as for_review")
+        .where("review_date < ?", Time.now)
```

Comment:
Place the . on the previous line, together with the method call receiver.<br>Use 2 (not 4) spaces for indenting an expression spanning multiple lines.<br>Do not use `Time.now` without zone. Use one of `Time.zone.now`, `Time.now.current`, `Time.now.in_time_zone`, `Time.now.utc`, `Time.now.getlocal`, `Time.now.iso8601`, `Time.now.jisx0301`, `Time.now.rfc3339`, `Time.now.to_i`, `Time.now.to_f` instead.